### PR TITLE
update benchmark, ms version of example/stable_diffusion_v2 docs and readme.

### PR DESCRIPTION
**Pull request by LiTingyu1997**
# What does this PR do?

1. update benchmark for ms2.3.1 and ms2.1.0
2. bugfix for Textual Inversion Fine-tuning
3. add train_config_lora_v2_vpred.yaml 

Fixes # (issue)

Adds # (feature)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/mindspore-lab/mindone/blob/master/CONTRIBUTING.md)?
- [ ] Did you make sure to update the documentation with your changes? E.g. record bug fixes or new features in `What's New`. Here are the
      [documentation guidelines](https://github.com/mindspore-lab/mindcv/wiki/%E6%96%87%E6%A1%A3%E7%BC%96%E5%86%99%E8%A7%84%E8%8C%83)
- [ ] Did you build and run the code without any errors?
- [ ] Did you report the running environment (NPU type/MS version) and performance in the doc? (better record it for data loading, model inference, or training tasks)
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@xxx

### CODE REVIEW

File: examples/stable_diffusion_v2/docs/en/image_variation_unclip.md
Comment (SamitHuang) [inline examples/stable_diffusion_v2/docs/en/image_variation_unclip.md:1]: why remove image variation?

File: examples/stable_diffusion_v2/docs/en/dreambooth_finetune.md [hunk 1]
```diff
@@ -297,9 +259,40 @@ Now, set `--ckpt_path` as `models/toonyou_beta3.ckpt` and run `text_to_image.py`
   <em> Figure 8. The generated images of the ToonYou (beta3) DreamBooth model using the prompt 'A girl, cherry blossoms, pink flowers, spring season'. </em>
 </p>
 
+## Evaluation
```
Comment (CaitinZhao) [inline examples/stable_diffusion_v2/docs/en/dreambooth_finetune.md:262]: 这个是不是放外面的readme？

File: examples/stable_diffusion_v2/docs/en/faq.md [hunk 2]
```diff
@@ -1,26 +0,0 @@
-# FAQ - Frequently Asked Questions
```
Comment (CaitinZhao) [inline examples/stable_diffusion_v2/docs/en/faq.md:1]: 这个为啥也删掉了

File: examples/stable_diffusion_v2/docs/en/inference_with_deliberate.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ Deliberate is a highly specialized Image generation AI Model of type Safetensors
 
 Take Deliberate_v2 as an example, this doc shows how to infer with the Deliberate model based on Mindone SD 1.5.
 
-### Step 1. Model conversion
+### Model conversion
```
Comment (CaitinZhao) [inline examples/stable_diffusion_v2/docs/en/inference_with_deliberate.md:8]: 这篇文档在外面的readme有链接吗?

File: examples/stable_diffusion_v2/docs/en/lora_finetune.md [hunk 4]
```diff
@@ -196,21 +193,6 @@ python eval/eval_clip_score.py --image_path_or_dir {path/to/generated_images} --
 ```
 
 For details, please refer to the guideline [Diffusion Evaluation](tools/eval/README.md).
-
-Here are the evaluation results for our implementation.
```
Comment (CaitinZhao) [inline examples/stable_diffusion_v2/docs/en/lora_finetune.md:200]: 这个为啥删掉？

File: examples/stable_diffusion_v2/docs/en/v_prediction.md [hunk 5]
```diff
@@ -200,18 +187,6 @@ python eval/eval_clip_score.py --image_path {path/to/test_images} --prompt {path
 
 For details, please refer to the guideline [Diffusion Evaluation](file:///C:/01Code/mindone/examples/stable_diffusion_v2/eval/README.md).
 
-Here are the evaluation results for our implementation, where the batch size is `1` and finetune method is vanilla for all experiments.
```
Comment (CaitinZhao) [inline examples/stable_diffusion_v2/docs/en/v_prediction.md:203]: 同上

File: examples/t2i_adapter/README.md [hunk 6]
```diff
@@ -31,29 +31,34 @@ There are multiple advantages of this architecture:
...
-|                  | OpenPose      |       1.4        | LAION-Aesthetics V2 (625K)             |                                    | [Download](https://download.mindspore.cn/toolkits/mindone/t2i-adapters/t2iadapter_openpose_sd14v1-ebcdb5cb.ckpt) |
-|                  | Segmentation  |       1.4        | [COCO-Stuff](#segmentation-coco-stuff) |                                    | [Download](https://download.mindspore.cn/toolkits/mindone/t2i-adapters/t2iadapter_seg_sd14v1-1d2e8478.ckpt)      |
-|                  | Sketch        |       1.5        | [COCO-Stuff](#segmentation-coco-stuff) |                                    | [Download](https://download.mindspore.cn/toolkits/mindone/t2i-adapters/t2iadapter_sketch_sd15v2-6c537e26.ckpt)   |
-|                  | Style         |       1.4        |                                        |                                    | [Download](https://download.mindspore.cn/toolkits/mindone/t2i-adapters/t2iadapter_style_sd14v1-a620ae97.ckpt)    |
+SDXL
+| task          | sd train version | dataset                                | weights                                                                                                          |
```
Comment (CaitinZhao) [inline examples/t2i_adapter/README.md:38]: 这个是不是不用更新了，整个t2i_adaper需要重构

File: examples/t2i_adapter/README.md
Comment (SamitHuang) [inline examples/t2i_adapter/README.md:1]: 建议t2i adapter的改写单放一个PR

File: examples/stable_diffusion_v2/docs/en/image_variation_unclip.md
Comment (LiTingyu1997) [reply examples/stable_diffusion_v2/docs/en/image_variation_unclip.md:1]: 升级2.3.0时未进行测试验证，2.3.1推理为黑图。版本无法保证与主页一致，先删除。

File: examples/stable_diffusion_v2/docs/en/faq.md (hunk 2 above)
Comment (LiTingyu1997) [reply examples/stable_diffusion_v2/docs/en/faq.md:1]: 回答的不是用户主要有问题的地方，为避免吐槽先删除。

File: examples/stable_diffusion_v2/docs/en/dreambooth_finetune.md (hunk 1 above)
Comment (LiTingyu1997) [reply examples/stable_diffusion_v2/docs/en/dreambooth_finetune.md:262]: lora/dreambooth/textual-inversion统一放里面

File: examples/stable_diffusion_v2/docs/en/inference_with_deliberate.md (hunk 3 above)
Comment (LiTingyu1997) [reply examples/stable_diffusion_v2/docs/en/inference_with_deliberate.md:8]: 没有

File: examples/stable_diffusion_v2/docs/en/lora_finetune.md (hunk 4 above)
Comment (LiTingyu1997) [reply examples/stable_diffusion_v2/docs/en/lora_finetune.md:200]: 升级2.3.0时未进行测试验证，版本无法保证与主页一致，删除单一版本的性能/精度数据。

File: examples/t2i_adapter/README.md (hunk 6 above)
Comment (LiTingyu1997) [reply examples/t2i_adapter/README.md]: 已移除

**Review by CaitinZhao — APPROVED** [review]

File: examples/t2i_adapter/README.md [hunk 7]
```diff
@@ -367,17 +361,18 @@ python adapter_image2image_sd.py \
...
-experiments are tested on ascend 910 with mindspore 2.1 graph mode
 
 <div align="center">
 
-| Task         | SD Version | Dataset                                      | cards| Train Time      | Throughput   | Recipe                             |
+| Task         | SD Version | Dataset                                      | Context         | Train Time      | Throughput   | Recipe                             |
 |--------------|:----------:|----------------------------------------------|-----------------|-----------------|--------------|------------------------------------|
-| Segmentation |    2.1     | [COCO-Stuff Train](#segmentation-coco-stuff) | 4 | 10h 35m / epoch | 39.2 img / s | [yaml](configs/sd_v2.1_train.yaml) |
-
+| Segmentation |    2.1     | [COCO-Stuff Train](#segmentation-coco-stuff) | D910Ax4-MS2.1-G | 10h 35m / epoch | 39.2 img / s | [yaml](configs/sd_v2.1_train.yaml) |
```
Comment (SamitHuang) [inline examples/t2i_adapter/README.md:369]: remove context?

**Review by SamitHuang — APPROVED** [review]

**Pull request merged**